pyaerocom/colocateddata.py

```diff
@@ -1878,20 +1885,20 @@ def plot_coordinates(self, marker="x", markersize=12, fontsize_base=10, **kwargs
             **kwargs,
         )
 
-    def __contains__(self, val):
-        return self.data.__contains__(val)
+    # def __contains__(self, val):
+    #     return self.data.__contains__(val)
 
-    def __repr__(self):
-        tp = type(self).__name__
-        dstr = "<empty>" if self._data is None else repr(self._data)
-        return f"pyaerocom.{tp}: data: {dstr}"
+    # def __repr__(self):
+    #     tp = type(self).__name__
+    #     dstr = "<empty>" if self.data is None else repr(self.data)
+    #     return f"pyaerocom.{tp}: data: {dstr}"
 
-    def __str__(self):
```

Comment:
Is this commented code still needed?

Reply:
`BaseModel` has a `__str__` method, which this class would inheret, but I agree this overloading is probably what we want.